packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -568,6 +568,7 @@ const TagSubforumPage2 = ({classes}: {
   const commentNodeProps = {
     treeOptions: {
       postPage: true,
+      showPostTitle: false,
```

Comment:
This is to make sure the "Biosecurity Subforum" label wasn't appearing at the top of threads in the subforum. We might need to make sure that it doesn't cause issues with https://github.com/ForumMagnum/ForumMagnum/pull/6350